test/ibmq/test_ibmq_client_v2.py

```diff
@@ -298,3 +231,163 @@ def test_url_unreachable(self, qe_token, qe_url):
+        self.assertIn('backend', self.job)
+        job_excluded = self.client.get_job(self.job_id, exclude_fields=['backend'])
+
+        # Ensure the result only excludes the specified field
+        self.assertNotIn('backend', job_excluded)
+        self.assertIn('shots', self.job)
+
+    def test_get_job_includes_nonexistent(self):
+        """Check get_job including nonexistent fields."""
+        # Get the job, including an nonexistent field.
+        self.assertNotIn('dummy_include', self.job)
+        job_included = self.client.get_job(self.job_id, include_fields=['dummy_include'])
+        # Ensure the result is empty, since no existing fields are included
+        self.assertFalse(job_included)
+
+    def test_get_job_excludes_nonexistent(self):
+        """Check get_job excluding nonexistent fields."""
+        # Get the job, excluding an non-existent field.
+        self.assertNotIn('dummy_exclude', self.job)
+        self.assertIn('shots', self.job)
+        job_excluded = self.client.get_job(self.job_id, exclude_fields=['dummy_exclude'])
+
+        # Ensure the result only excludes the specified field. We can't do a direct
+        # comparison against the original job because some fields might have changed.
+        self.assertIn('shots', job_excluded)
+
+    def test_job_submit(self):
+        """Test job submission."""
+        result = self.client.job_submit(self.backend_name, self.qobj.to_dict())
+        status_present = ['status' in [dict_ for dict_ in result.get('qasms')]]
+        self.assertTrue(status_present)
+
+    @skip
+    # TODO: Q-Object-External-Storage property is not allowed in this backend
+    def test_job_submit_object_storage(self):
+        """Test job submission using object storage."""
+        # Get the job via object storage.
+        # result = self.client.job_submit_object_storage(self.backend_name,
+        #                                                self.qobj.to_dict())
+
+        # TODO
+        pass
+
+    @skip
+    # TODO: RequestsApiError: 404 Client Error: Not Found for url: [...] None, Error code: NotFound.
+    def test_job_download_qobj_object_storage(self):
+        """Test job Qobj download using object storage."""
+        # result = self.client.job_download_qobj_object_storage(self.job_id)
+
+        # TODO
+        pass
+
+    @skip
+    # TODO: RequestsApiError: 404 Client Error: Not Found for url: [...] None, Error code: NotFound.
+    def test_job_result_object_storage(self):
+        """Test getting a job's result using object storage."""
+        # result = self.client.job_result_object_storage(self.job_id)
+
+        # TODO
+        pass
+
+    def test_job_get_responsive(self):
+        """Test getting a job responds with something other than None."""
+        result = self.client.job_get(self.job_id)
+        status_present = ['status' in [dict_ for dict_ in result.get('qasms')]]
+        self.assertTrue(status_present)
+
+    def test_job_get_filtered_fields(self):
```

Comment:
Can you check if this tests is not already covered by the moved `test_get_job_includes_xxx/test_get_job_excludes_xxx` tests a few lines above? Note that `client.get_job()` is actually an alias for `client.job_get()` for compatibility purposes (actually, we should rename the mentioned tests to `test_job_get_includes_....` for consistency).

Reply:
Those tests test `get_job_xxx`, not `job_get_xxx`. I can update them to do the latter, or if we need both tests I'll keep the similar ones currently in the PR.